app/jobs/calendar_importer/event_resolver.rb

```diff
@@ -214,7 +214,7 @@ def save_all_occurences
 
       event_time[:are_spaces_available] = occurence.status if occurence.respond_to?(:status)
 
-      unless event.update(data.attributes.merge(event_time))
+      unless event.update!(data.attributes.merge(event_time))
```

Comment:
Doesn't this mean we never get errors reported to the user because the `!` makes it cause an exception instead? Why did we make this change?